gateway/src/main.rs

```diff
@@ -138,12 +138,20 @@ async fn init(db: SqlitePool, args: InitArgs) -> io::Result<()> {
         None => Key::new_random(),
     };
 
-    query("INSERT INTO accounts (account_name, key, super_user) VALUES (?1, ?2, 1)")
-        .bind(&args.name)
-        .bind(&key)
-        .execute(&db)
-        .await
-        .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
+    query(
+        r#"
+    INSERT INTO accounts (account_name, key, super_user) 
+    VALUES (?1, ?2, 1)
+    ON CONFLICT (account_name)
+    DO
+        UPDATE SET key = ?2
```

Comment:
This might be a bit dangerous. It means that anyone who gains unauthorized access to the VM will be able to cycle any key to their choosing. Maybe we should rather drop the `ensure_success` when creating the user in e2e (and also change the username to `test`)?

Reply:
That's a very good point, thanks Pieter! I'll implement your suggestion.